tokio/src/task/mod.rs

```diff
@@ -1,5 +1,202 @@
 //! Asynchronous green-threads.
-
+//!
+//! ## What are Tasks?
+//!
+//! A _task_ is a light weight, non-blocking unit of execution. A task is similar
+//! to an OS thread, but rather than being managed by the OS scheduler, they are
+//! managed by the [Tokio runtime][rt]. Key points about tasks include:
```

Comment:
This section is great! I'd note somewhere that Tokio's tasks are similar to goroutines. Something along the lines of:

- If you are familiar with [Go's goroutines](https://tour.golang.org/concurrency/1), [Kotlin's coroutines](https://kotlinlang.org/docs/reference/coroutines-overview.html), or [Erlang's processes](http://erlang.org/doc/getting_started/conc_prog.html#processes), you can think of Tokio's tasks as something similar.

Reply:
Done! https://github.com/tokio-rs/tokio/pull/1801/commits/467d4dcda961e882ac7c8434bdfc59620c048b40 Thanks for the suggestion!